baseballstadium/src/components/dashboard.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+class Dashboard extends React.Component {
+  state = {
+    players: [{ id: 1, name: 'Pedro Martinez' }, { id: 2, name: 'Kai Lopez' }],
+    playerAtBat: 'Pedro Matinez',
+    strikesAndFouls: 0,
+    balls: 0
+  };
+
+  getPlayerIndex = playerName => {
+    this.state.players.forEach((player, index) => {
+      if (player.name === playerName) {
+        return index;
+      }
+    });
+  };
+
+  nextPlayer = playerName => {
```

Comment:
I like that you're keeping your state and functions in the Dashboard component, makes sense.